CONTRIBUTING.md

```diff
@@ -10,25 +10,22 @@ We welcome contributions to `guidance`, and this document exists to provide usef
 
 ## Developer Setup
 
-The quickest way to get started is to run (in a fresh environment):
+Start by creating a fresh environment that includes a rust compiler with something like:
 ```bash
-pip install -e .[all,test,bench]
+conda create --name guidancedev python=3.12 rust
```

Comment:
Do we even need the _rust directory and rust compiler anymore now that llguidance is independenly published? Was thinking of just deleting that subdirectory for simplicity

Reply:
maybe not.  will test

Reply:
rust not needed anymore (see also https://github.com/guidance-ai/guidance/pull/1140)